### ISSUE TITLE
No head-object support for the s3api command?

### ISSUE THREAD

**Issue by mzy2240** [sengaya/aws-micro-cli#11]
Per subject. Wondering if it is possible to support query meta data for objects.

**Comment by sengaya** [comment]
I haven't worked on this project for a while but I may take a look at it.
Could you provide the respective aws cli command and some sample output?

**Comment by mzy2240** [comment]
Sure that would be great!
The command is `$ aws s3api head-object --bucket mybucket --key path/to/myfile.csv --query "ContentLength"` and it returns the number of bytes (integer) of the object.

**Comment by sengaya** [comment]
I added support for `s3api head-object`. I do not want to support the `--query` option for now ( but PR welcome 😉). It should be easy to cover your use case e.g. by parsing the output with `jq` or a combination of `grep` and `sed`.

So far I could test it only with this public s3 bucket:
```
aws-micro s3api head-object --no-sign-request --bucket fah-public-data-covid19-cryptic-pockets --key human/ace2/PROJ14532/RUN1880/CLONE0/results95/frame95.xtc
```
Could you test it on your side as well and report back? Check out the branch [actions-update_s3api-head-object](https://github.com/sengaya/aws-micro-cli/tree/actions-update_s3api-head-object).

**Comment by mzy2240** [comment]
yeah it works great! thank you! do you mind to merge to the main branch and do a release? thanks!

**Comment by sengaya** [comment]
Thanks for the feedback! v0.4.0 was just released 🚀
